tests/testthat/test-maic_anchored.R

```diff
@@ -255,18 +145,18 @@ test_that("maic_anchored for binary case gives the expected result", {
 
   expect_equal(
     result$inferential$report_overall_robustCI$`OR[95% CI]`,
-    c("1.70[1.28;2.26]", "", "1.46[0.85;2.51]", "", "2.33[1.75;3.12]", "", "0.63 [0.34; 1.16]")
+    c("1.70[1.28;2.26]", "", "1.14[0.67;1.95]", "", "2.33[1.75;3.12]", "", "0.49 [0.27; 0.90]")
   )
   expect_equal(
     result$inferential$report_overall_robustCI$`n.events(%)`,
-    c("390(78.0)", "338(67.6)", "128.8(25.8)", "115.1(23.0)", "280(58.3)", "120(37.5)", "--")
+    c("390(78.0)", "338(67.6)", "128.8(25.8)", "124.2(24.8)", "280(58.3)", "120(37.5)", "--")
   )
   expect_equal(
     result$inferential$report_overall_robustCI$N,
     c("500", "500", "500", "500", "480", "320", "--")
   )
   expect_equal(
     result$inferential$report_overall_bootCI$`OR[95% CI]`,
-    c("1.70[1.28;2.26]", "", "1.46[0.63;1.58]", "", "2.33[1.75;3.12]", "", "0.63 [0.27; 0.68]")
+    c("1.70[1.28;2.26]", "", "1.14[0.33;0.98]", "", "2.33[1.75;3.12]", "", "0.49 [0.14; 0.42]")
```

Comment:
@MikeJSeo with this new data, the bootstrap has worse bias than the previous!
` "0.63 [0.27; 0.68]"` vs `"0.49 [0.14; 0.42]"`